### PR TITLE
Pyfluent example updates w.r.t. documentation and API's

### PR DESCRIPTION
**Pull request by abhishekchitwar**
Pyfluent example updates as follows:

- Updated documentation using Jupytext.
- Pyfluent API update for connecting fluent server.

### CODE REVIEW

File: examples/pyfluent-mixing-elbow/main.py [hunk 1]
```diff
@@ -1,176 +1,154 @@
-# # Fluent setup and solution using PyFluent APIs
-# ------------------------------------------------
+# # Integrating PyFluent with PyWorkbench: 
+# This example showcases how to use PyFluent workflow together with PyWorkbench - (Python client scripting for Ansys Workbench).
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:2]:
```suggestion
# # Integrating PyFluent with PyWorkbench 
# This example shows how to use the PyFluent workflow with PyWorkbench - (Python client scripting for Ansys Workbench).
```
It's unclear what the content in the parentheses is supposed to convey. Can you clarify, breaking it into two sentences.

File: examples/pyfluent-mixing-elbow/main.py [hunk 2]
```diff
@@ -1,176 +1,154 @@
...
+#
 # This example sets up and solves a three-dimensional turbulent fluid flow
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:4]:
```suggestion
# This example sets up and solves a 3D turbulent fluid flow
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 3]
```diff
@@ -1,176 +1,154 @@
...
 # and heat transfer problem in a mixing elbow, which is common in piping
 # systems in power plants and process industries. Predicting the flow field
 # and temperature field in the area of the mixing region is important to
 # designing the junction properly.
 #
-# This example uses settings objects.
+# This example uses Pyfluent settings objects API's.
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:10]:
```suggestion
# This example uses Pyfluent settings objects APIs.
```
The plural form has no apostrophe.

File: examples/pyfluent-mixing-elbow/main.py [hunk 4]
```diff
@@ -1,176 +1,154 @@
...
 import os
 import pathlib
 from ansys.workbench.core import launch_workbench
-
 import ansys.fluent.core as pyfluent
 from ansys.fluent.core import examples
 from ansys.fluent.visualization.pyvista import Graphics
 
-# # Specify client and server directories and launch WB service (This example launches WB locally)
+# ## Specify client and server directories with launch WB service.
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:30]:
```suggestion
# ## Specify client and server directories and launch a Workbench service.
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 5]
```diff
@@ -1,176 +1,154 @@
...
 
-# # Get the input file from example data and upload to server directory
+# ## Download the input file from example data and upload to server directory.
 
 import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
 wb.upload_file(import_filename)
 
-# +
-# run a Workbench script to define the project and load geometry
+# ## Generate a "FLUENT" System using Ansys Workbench Scripting API (used for Journaling) and parse it to the PyWorkbench API.
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:40]:
```suggestion
# ## Generate a "FLUENT" system using the Ansys Workbench Scripting API, which is used for journaling. Then, parse it using the PyWorkbench API.
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 6]
```diff
@@ -1,176 +1,154 @@
...
 wb.run_script_string('system1 = template1.CreateSystem()')
 
-# -
 
-###############################################################################
-# # Launch Fluent
-# # ~~~~~~~~~~~~~
-# # Launch Fluent as a service in meshing mode with double precision running on
-# # two processors.
+# ## Launch Fluent & Connect to Fluent
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:48]:
```suggestion
# ## Launch Fluent and connect to it
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 7]
```diff
@@ -1,176 +1,154 @@
...
+# Launch Fluent as server with PyWorkbench API and and connect to Pyfluent session
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:49]:
```suggestion
# Launch Fluent as a server with the PyWorkbench API and then connect to a PyFluent session.
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 8]
```diff
@@ -1,176 +1,154 @@
...
-# # Specification Method: Intensity and Hydraulic Diameter
-# # Turbulent Intensity: 5 [%]
-# # Hydraulic Diameter: 4 [inch]
-# # Temperature: 293.15 [K]
+# ## Set up boundary conditions for CFD analysis
 
-cold_inlet = fluent_session.setup.boundary_conditions.velocity_inlet["cold-inlet"]
+# Set up the boundary conditions for the inlets, outlet, and walls for CFD
+# analysis.
+# - cold inlet (cold-inlet), Setting: Value:
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:98]:
```suggestion

# - cold inlet (cold-inlet), Setting: Value:
```
A blank line is needed before the list of items to render correctly in the doc.

File: examples/pyfluent-mixing-elbow/main.py [hunk 9]
```diff
@@ -1,176 +1,154 @@
...
+
+fluent_session.solution.initialization.hybrid_initialize()
+
+# ## Set iterations for 
+# Setting iteration count to 150 for solving the model.
 
 fluent_session.solution.run_calculation.iter_count = 150
 
-# # Update Solution using WB Journal Command
+# ## Update Solution using Workbench Journal Commands
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:144]:
```suggestion
# ## Update solution using Workbench journal commands
```
Headings are in sentence case. Only capitalize proper nouns.

File: examples/pyfluent-mixing-elbow/main.py [hunk 10]
```diff
@@ -1,176 +1,154 @@
...
-###############################################################################
-# # Create velocity vectors
-# # ~~~~~~~~~~~~~~~~~~~~~~~
-# # Create and display velocity vectors on the ``symmetry-xyplane`` plane
-
-###############################################################################
-# # Post processing with PyVista (3D visualization)
-# # =============================================
+# ## Post processing - Create velocity vectors
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:151]:
```suggestion
# ## Postprocess
# Create and display velocity vectors on the ``symmetry-xyplane`` plane.
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 11]
```diff
@@ -197,22 +173,23 @@
...
 
-# # Save project
+# ## Save project
 
 file_path = os.path.join(workdir, "mixing_elbow.wbpj")
 save_string = "Save(FilePath=\"" + file_path + "\"," + "Overwrite=True)"
-# wb.run_script_string('Save(FilePath="mixing_elbow1.wbpj", Overwrite=True)')
 wb.run_script_string(save_string)
 
+# ## Archive Project
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:182]:
```suggestion
# ## Archive project
```

File: examples/pyfluent-mixing-elbow/main.py [hunk 12]
```diff
@@ -197,22 +173,23 @@
...
 archive_string = "Archive(FilePath=\"" + file_path + "\")"
 wb.run_script_string(archive_string)
 
-# # Download the WB archived project
+# ## Download the archived project
 
 wb.download_file("mixing_elbow.wbpz")
 
-fluent_session.exit()
+# ## Exit Fluent & Workbench Sessions.
```
Comment (PipKat) [inline examples/pyfluent-mixing-elbow/main.py:192]:
```suggestion
# ## Exit Fluent and Workbench sessions
```
Given that this is a heading, no punction.

**Review by PipKat — COMMENTED** [review]
@abhishekchitwar I started working on edits to the PyWorkbench examples locally. I'd like to see this and other doc PR merged ASAP so that I can pull the changes into my branch. Would you be able to get the checks running and this PR merged early next week?

**Comment by franknli** [comment]
@abhishekchitwar Do you still need this pull request? Or you can close it.

**Comment by abhishekchitwar** [comment]
Closing since all updated are merged with PR #35. Thanks!

**Pull request closed without merging**